modules/end-user-guide/pages/user-onboarding.adoc

```diff
@@ -7,8 +7,9 @@
 [id="user-onboarding_{context}"]
 = User onboarding
 
-If your organization is already running a {prod-short} instance, you can get started as a new user of {prod-short} by first learning how to start a new workspace and how to manage your workspaces:
+If your organization is already running a {prod-short} instance, you can get started as a new user by learning how to start a new workspace, manage your workspaces, and authenticate yourself to remote Git repositories from workspaces:
 
 . xref:starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc[]
 . xref:optional-parameters-for-the-urls-for-starting-a-new-workspace.adoc[]
 . xref:basic-actions-you-can-perform-on-a-workspace.adoc[]
+. xref:authenticating-yourself-to-a-git-server-from-a-workspace.adoc[]
```

Comment:
Suggestion: The xref says authenticate yourself to a git server and the lead sentence says remote Git repositories. Stick to one type of usage for consistency.

Reply:
Thank you so much for catching this, Sir!